src/pages/ni/customers/CustomersFundingsMonitoring.vue

```diff
@@ -112,9 +129,36 @@ export default {
       ],
     };
   },
+  computed: {
+    displayedCustomersFundingsMonitoring () {
+      let allCustomersFundingsMonitoring = this.allCustomersFundingsMonitoring;
+      if (this.selectedSector !== '') {
+        allCustomersFundingsMonitoring = allCustomersFundingsMonitoring.filter(elem =>
+          this.$_.get(elem, 'sector._id', null) === this.selectedSector);
+      }
+      if (this.selectedThirdPartyPayer !== '') {
+        allCustomersFundingsMonitoring = allCustomersFundingsMonitoring.filter(elem => {
+          return elem.tpp._id === this.selectedThirdPartyPayer
+        });
+      }
+      return allCustomersFundingsMonitoring;
+    },
+  },
+  methods: {
+    async getThirdPartyPayerOptions () {
+      try {
+        const thirdPartyPayerOptions = (await this.$thirdPartyPayers.list()).map(elem => { return { value: elem._id, label: elem.name } });
+        thirdPartyPayerOptions.push({ value: '', label: 'Tous les financeurs' });
+        return thirdPartyPayerOptions;
+      } catch (e) {
+        this.thirdPartyPayerOptions = [];
+      }
+    },
+  },
   async mounted () {
     try {
       this.tableLoading = true;
+      this.thirdPartyPayerOptions = await this.getThirdPartyPayerOptions();
```

Comment:
il vaut mieux que tu initialises la variable `this.thirdPartyPayerOptions` dans la methode `await this.getThirdPartyPayerOptions()`